src/many-abci/src/module.rs

```diff
@@ -34,7 +36,9 @@ fn _many_block_from_tendermint_block(block: tendermint::Block) -> Block {
                 id: TransactionIdentifier {
                     hash: hasher.finalize().to_vec(),
                 },
-                content: Some(b),
+                content_: None,
+                request: minicbor::decode(b.as_ref()).ok(),
```

Comment:
We should not probably not ignore the error here

Reply:
@fmorency Do you want me to make `_many_block_from_tendermint_block` return a result here where deserialization fails, instead of setting the request to `None` when that happens?